.ci/azure/linux_arm64.yml

```diff
@@ -17,13 +21,31 @@ jobs:
     system.debug: true
     VSTS_HTTP_RETRY: 5
     VSTS_HTTP_TIMEOUT: 200
+    PYTHON_ARM_VERSION: "3.8.12"
+    OPENVINO_ARCH: 'aarch64'
+    NUM_PROC: 1
     BUILD_TYPE: Release
-    REPO_DIR: $(Build.Repository.LocalPath)
-    OPENVINO_REPO_DIR: $(REPO_DIR)/../openvino
-    OPENCV_REPO_DIR: $(REPO_DIR)/../opencv
+    REPO_OPENVINO: $(REPO_CONTRIB)/../openvino
+    REPO_CONTRIB: $(Build.Repository.LocalPath)
+    REPO_OPENCV: $(REPO_CONTRIB)/../opencv
+    REPO_OMZ: $(REPO_CONTRIB)/../open_model_zoo
```

Comment:
OMZ is a part of openvino repo, use submodule

Reply:
done